package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file and the corresponding `package-lock.json` shouldn't exist, we have those in the client directory